### ISSUE TITLE
create a pathway commons brick

### ISSUE THREAD

**Issue by tomlue** [biobricks-ai/pathwaycommons#1]
1. Use the [bricktools::check](https://github.com/biobricks-ai/bricktools) function @achatrath is this usable now?
2. Emulate the brick on other bricks (like github.com/biobricks-ai/hello-brick)
3. Ask clarifying questions on slack

In the end, these commands should work:
```R
library(biobricks)
install_brick("pathwaycommons")
brick_pull("pathwaycommons")
brick_load_arrow("pathwaycommons")
```
Looks like the download files are at
https://www.pathwaycommons.org/archives/PC2/?C=M;O=D

**Comment by achatrath** [comment]
Yes, it is functional but currently only does a few basic checks for brick structure (eg, dvc.yaml, dvc.lock, .dvc/config, README.md present in the expected locations of the directory and not empty). 

The function can be called using:

bricktools::check_brick(dir) 

where dir is the name of the directory with the brick.

I named the function check_brick to not be confused with devtools::check though this is easy to change if we prefer bricktools::check
